ddl/ddl_worker.go

```diff
@@ -1066,6 +1090,7 @@ func updateSchemaVersion(t *meta.Meta, job *model.Job) (int64, error) {
 				diff.AffectedOpts = buildPlacementAffects(oldIDs, oldIDs)
 			}
 		}
+	case model.ActionRecoverSchema:
```

Comment:
Please add some comments for it.

Reply:
I can't figure out why you do nothing here, but the `flashback database` also can work.

Reply:
Oh, I got you, brilliant, please add some comment here to explain how `infoschema/builder.go(ApplyDiff)` to load schema for this kind of ddl job.